com.corifeus.onenote.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.corifeus.onenote</id>
+  
+  <name>P3X OneNote</name>
+  <summary>📚 Linux Electron Onenote - A Linux compatible version of OneNote </summary>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      This is a Liunx OneNote helper program. For verbose description refer to the GitHub below...
```

Comment:
This really should be expanded upon.

Reply:
Upstream doesn't provide more.

I've just copied from their snap page (https://snapcraft.io/p3x-onenote#cory-non-external)

Reply:
We're not Snap though. There's a better description at https://www.corifeus.com/onenote.